.changeset/mighty-singers-prove.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/flight-icons": minor
+---
+
+Add static SVG sprite sheet file
```

Comment:
nitpick: we've just updated [our changelog guidance](https://github.com/hashicorp/design-system/blob/main/wiki/Website-Changelog.md#general-guidance) to use past tense in changelog entries
```suggestion
Added static SVG sprite sheet file
```